packages/bpk-component-card-list/src/BpkCardListRowRail/BpkCardListCarousel.module.scss

```diff
@@ -17,27 +17,33 @@
  */
 
 @use '../../../unstable__bpk-mixins/tokens';
+@use '../../../unstable__bpk-mixins/breakpoints';
+@use '../../../unstable__bpk-mixins/utils';
 
 .bpk-card-list-row-rail {
   &__row,
   &__rail {
     display: flex;
     padding-top: tokens.bpk-spacing-sm();
     padding-bottom: tokens.bpk-spacing-sm();
-    overflow-x: scroll;
+    overflow-x: hidden;
```

Comment:
In desktop, scroll card by wheel/touchpad will change [currentIndex](https://github.com/Skyscanner/backpack/pull/3878/files#diff-b442a953b34523f87814a56f06504909e6ea34b995d5e1965d4e3359c92417cdL83), And pagination will also change the currentIndex. It is rather difficult to keep the two in perfect synchronization, and the functionality is redundant with that of pagination so disable this feature.